codalab/model/bundle_model.py

```diff
@@ -2475,18 +2476,27 @@ def add_user(
         :param affiliation:
         :return: (new integer user ID, verification key to send)
         """
+
+        def is_academic_email(email):
+            """
+            This is a basic function that can be used to compare the email domain suffix with a list of academic email domains.
+            Academic emails typically have domains such as "yy.edu" or "xyz.edu.xx" (where "edu" is followed by a country code).
+            """
+            email_suffix = EDU_USER_REGEXS.findall(email.lower())
+            return len(email_suffix) > 0
+
         with self.engine.begin() as connection:
             now = datetime.datetime.utcnow()
             user_id = user_id or '0x%s' % uuid4().hex
 
             time_quota = (
                 self.default_user_info['edu_time_quota']
-                if email.endswith(".edu")
+                if is_academic_email(email)
```

Comment:
Can we add some unit tests for is_academic_email?

Reply:
Thanks Ashwin! For consistency, I add this unit test to bundle_model_test for now.